backend_new/package.json

```diff
@@ -38,6 +38,8 @@
     "class-validator": "^0.14.0",
     "class-transformer": "^0.5.1",
     "cloudinary": "^1.37.3",
+    "dayjs": "^1.11.9",
+    "jwt-simple": "^0.5.6",
```

Comment:
Is this package still maintained? The last publish was four years ago. Are there alternatives that are still active? I know it doesn't mean it's better,  [jsonwebtoken](https://www.npmjs.com/package/jsonwebtoken) looks to be active and has way more weekly downloads.

Reply:
Yeah I was looking at it, it seemed stable, but I think you're right

switched to the new package!